mitodl/ocw-hugo-themes
# Added download button to control bar

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #731
 
#### What's this PR do?
- Removes the download video and download download transcript button.
- Adds download button to control bar. 

#### How should this be manually tested?
- Starts video player.
- On control bar click download button
- Verify menu opens
- Click menu item and verify the functionality is working

#### Screenshots (if appropriate)
<img width="1438" alt="Screenshot 2022-06-08 at 4 37 22 AM" src="https://user-images.githubusercontent.com/87968618/172501262-25111cd5-7fe1-4f1a-abc8-be53c341fd7c.png">
<img width="1438" alt="Screenshot 2022-06-08 at 4 37 17 AM" src="https://user-images.githubusercontent.com/87968618/172501290-5b29fa11-2478-44ed-ae3b-8f6b2cba0efd.png">
